node-cli/config.js

```diff
@@ -1 +1 @@
-exports.realmAppId = "TODO";
+exports.realmAppId = "tasktracker-huhcb";
```

Comment:
Don't forget to stub this out before you merge!

Reply:
It should be fine as long as we're on the phase2 feature branch.